DROD/HoldSelectScreen.cpp

```diff
@@ -958,7 +958,6 @@ void CHoldSelectScreen::OnBetweenEvents()
 		// Done downloading the list.
 		CScreen::HideStatusMessage();
 		this->bDownloadList = false;
-		PopulateHoldListBox();
```

Comment:
How is the hold list populated now that this is removed?

Reply:
PopulateHoldList was being called twice.  This one was superfluous.

Reply:
I must be being dense. I'm not seeing the second call. Would you point it out to me?

Reply:
Okay.  I believe this is correct now.  It *does* re-populate the list here, but now selects the hold in question in this function, upon this repopulation.